synapse/logging/opentracing.py

```diff
@@ -737,24 +737,14 @@ def decorator(func):
 
             @wraps(func)
             async def _trace_inner(*args, **kwargs):
-                if opentracing is None:
+                with start_active_span(_opname):
                     return await func(*args, **kwargs)
```

Comment:
I tried to remove this earlier and it caused issues: https://github.com/matrix-org/synapse/pull/7872/#discussion_r455893714

I think the case this protects against is having `opentracing` installed, but disabled via the config.

Reply:
Did you also remove the explicit `scope.span.set_tag(...)`?

Reply:
Ah, no. That makes sense. 👍